### PR TITLE
PC-18 add end-point for payment conformation

### PR DESCRIPTION
**Pull request by Max43**

### CODE REVIEW

File: api_v2.json [hunk 1]
```diff
@@ -1732,11 +1733,38 @@
...
+							],
+							"port": "36016",
+							"path": [
+								"v2",
+								"payment",
+								"test",
+								"confirmation"
+							]
+						},
+						"description": "**GET /v2/payment/{id}/confirmation**\nPayment Confirmation API. It provides ability to get information about specific payment confiramiton entity, based on id.\n\n**Required Fields**\n\nid:Unique ID of payment confirmation\n\n**Response**\n\n  Id:Payment confirmation confirmation unique ID\n  \n  Short reference:the Payment confirmation short reference\n  \n  Status:Payment confirmation status\n  \n  Expires_at:Payment confirmation expiration date and time\n  \n  Confirmation_url: The url to the payment confirmation file - this should be available only when the status is 'completed'\n  \n  Account_id:The account id where the payment confirmation and the payment belong to\n  \n  Created_at:The date time when the confirmation was created\n  \n  Updated_at:The date time of the latest status update of the confirmation)"
```
Comment (oladan) [inline api_v2.json]: `confiramiton ` - typo

File: api_v2.json [hunk 2]
```diff
@@ -1732,11 +1733,38 @@
...
 					"response": []
+				},
+				{
+					"name": "Payment Confirmation",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {},
+						"url": {
+							"raw": "192.168.30.159:36016/v2/payment/test/confirmation",
```
Comment (oladan) [inline api_v2.json]: here should be some general `url` and `port`.

File: api_v2.json (hunk 1 above)
Comment (oladan) [reply api_v2.json]: also status can be `failed` or `completed`

Comment (oladan) [reply api_v2.json]: also we fetching payment_confirmation by payment id, not by payment_confirmation id

**Review by oladan — CHANGES_REQUESTED** [review]
please see comments

File: api_v2.json (hunk 1 above)
Comment (Max43) [reply api_v2.json]: thanks @oladan

File: api_v2.json [hunk 3]
```diff
@@ -1732,11 +1733,38 @@
...
+						"body": {},
+						"url": {
+							"raw": "{{url}}payment/{{payment_id}}/confirmation",
+							"host": [
+								"192",
+								"168",
+								"30",
+								"159"
+							],
+							"port": "36016",
```
Comment (oladan) [inline api_v2.json]: port and host should be general here also

File: api_v2.json [hunk 4]
```diff
@@ -1732,11 +1733,38 @@
...
+							"path": [
+								"v2",
+								"payment",
+								"test",
+								"confirmation"
+							]
+						},
+						"description": "**GET /v2/payment/{id}/confirmation**\nPayment Confirmation API. It provides ability to get information about specific payment confirmation entity, based on id.\n\n**Required Fields**\n\nid:Unique ID of payment\n\n**Response**\n\n  Id:Payment unique ID\n  \n  Short reference:the Payment confirmation short reference\n  \n  Status:Payment confirmation status\n  \n  Expires_at:Payment confirmation expiration date and time\n  \n  Confirmation_url: The url to the payment confirmation file - this should be available only when the status is 'completed'\n  \n  Account_id:The account id where the payment confirmation and the payment belong to\n  \n  Created_at:The date time when the confirmation was created\n  \n  Updated_at:The date time of the latest status update of the confirmation)"
```
Comment (oladan) [inline api_v2.json]: I believe it should be `v2/payments` not `v2/payment`

Comment (oladan) [reply api_v2.json]: you have `based on id` but `based on payment id` - sounds better

Comment (oladan) [reply api_v2.json]: you have `Id:Payment unique ID` - should be `Id:Payment confirmation unique ID`

File: api_v2.json [hunk 5]
```diff
@@ -1732,11 +1733,38 @@
...
 					"response": []
+				},
+				{
+					"name": "Payment Confirmation",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {},
+						"url": {
+							"raw": "{{url}}payment/{{payment_id}}/confirmation",
```
Comment (oladan) [inline api_v2.json]: here should be {{id}} which is payment id

File: api_v2.json (hunk 4 above)
Comment (oladan) [reply api_v2.json]: we also should have `payment_id` (which is payment id) in response

**Review by oladan — CHANGES_REQUESTED** [review]
please see comments

File: api_v2.json (hunk 5 above)
Comment (Max43) [reply api_v2.json]: @oladan call it payment_id to be more clear

Comment (oladan) [reply api_v2.json]: @Max43 in this context it should be {{id}} like in other samples for payment

Comment (Max43) [reply api_v2.json]: got it

**Pull request closed without merging**